src/program/services/scrapers/shared.py

```
@@ -24,20 +23,22 @@
 
 
 def _parse_results(
-    item: MediaItem, results: Dict[str, str], log_msg: bool = True
+    item: MediaItem,
+    results: Dict[str, str],
```

Ruff (0.14.6)

Undefined name `Dict`

(F821)